packages/nextjs/package.json

```diff
@@ -8,52 +8,51 @@
     "build": "next build",
     "serve": "next start",
     "lint": "next lint",
-    "format": "prettier --write . '!(node_modules|.next|contracts)/**/*'",
+    "format": "prettier --write . '!(node_modules|.next|contracts|out)/**/*'",
     "check-types": "tsc --noEmit --incremental",
     "vercel": "vercel",
     "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
   },
   "dependencies": {
-    "@heroicons/react": "^2.0.11",
-    "@rainbow-me/rainbowkit": "2.1.2",
-    "@tanstack/react-query": "^5.28.6",
-    "@uniswap/sdk-core": "^4.0.1",
-    "@uniswap/v2-sdk": "^3.0.1",
-    "blo": "^1.0.1",
+    "@heroicons/react": "^2.1.5",
+    "@rainbow-me/rainbowkit": "2.1.5",
+    "@tanstack/react-query": "^5.52.2",
+    "@uniswap/sdk-core": "^5.3.1",
+    "@uniswap/v2-sdk": "^4.4.1",
+    "blo": "^1.2.0",
     "burner-connector": "^0.0.8",
-    "daisyui": "4.5.0",
-    "next": "^14.0.4",
-    "next-themes": "^0.2.1",
+    "daisyui": "4.12.10",
+    "next": "^14.2.6",
+    "next-themes": "^0.3.0",
     "nprogress": "^0.2.0",
     "qrcode.react": "^3.1.0",
-    "react": "^18.2.0",
+    "react": "^18.3.1",
     "react-copy-to-clipboard": "^5.1.0",
-    "react-dom": "^18.2.0",
-    "react-hot-toast": "^2.4.0",
-    "use-debounce": "^8.0.4",
-    "usehooks-ts": "^2.13.0",
-    "viem": "2.17.4",
-    "wagmi": "2.10.10",
-    "zustand": "^4.1.2"
+    "react-dom": "^18.3.1",
+    "react-hot-toast": "^2.4.1",
+    "use-debounce": "^10.0.3",
+    "usehooks-ts": "^3.1.0",
```

Comment:
Ohh probably if usehooks works nicely now, we could remove use-debounce? since we have a debounce hook in usehookts.ts? 

https://usehooks-ts.com/react-hook/use-debounce-value